spec/avro_spec.rb

```diff
@@ -105,13 +107,108 @@
     define_schema "message.avsc", <<-AVSC
```

Comment:
* The fixture data is immediately apparent.
* Because fixtures aren't shared between examples, they're minimal and focused on demonstrating a single issue.